src/components/Button/Button.st.css

```diff
@@ -355,3 +358,55 @@
 .root:upgrade:size(large):mobile, .root:priority(primary):upgrade:size(large):mobile, .root:priority(basic):upgrade:size(large):mobile{
     padding: calculate(string(-), 17px, value(secondaryBorderWidth)) 16px;
 }
+
+.root .affixIcon svg path {
+    fill: none;
+    transition: stroke .2s ease-in-out;
+}
+
+.root .affixIcon svg path,
+.root:priority(basic) .affixIcon svg path,
+.root:priority(primary) .affixIcon svg path {
+    stroke: value(PrimaryTextColor);
+}
+
+.root:hover .affixIcon svg path,
+.root:priority(basic):hover .affixIcon svg path,
+.root:priority(primary):hover .affixIcon svg path {
+    stroke: applyOpacity(value(PrimaryTextColor), 0.7);
+}
```

Comment:
Here also, could we just define:
```
.root:hover .affixIcon svg path {
       stroke: applyOpacity(value(PrimaryTextColor), 0.7);
}
```

and then override only the `secondary` and the `basicSecondary` priority ?

Reply:
Good point 🤔